wrapper/src/main/java/software/amazon/jdbc/plugin/failover/FailoverConnectionPlugin.java

```diff
@@ -169,6 +189,14 @@ public FailoverConnectionPlugin(final PluginService pluginService, final Propert
     initSettings();
 
     this.staleDnsHelper = new AuroraStaleDnsHelper(this.pluginService);
+
+    TelemetryFactory telemetryFactory = this.pluginService.getTelemetryFactory();
+    this.failoverWriterTriggeredCounter = telemetryFactory.createCounter("writerFailover.triggered.count");
+    this.failoverWriterSuccessCounter = telemetryFactory.createCounter("writerFailover.completed.success.count");
+    this.failoverWriterFailedCounter = telemetryFactory.createCounter("writerFailover.completed.failed.count");
+    this.failoverReaderTriggeredCounter = telemetryFactory.createCounter("replicaFailover.triggered.count");
```

Comment:
Is this counter public facing? Will users see `replicaFailover`? Reader Failover is the more common terminology in logs and docs, if it is public facing I wonder if it makes sense to use readerFailover here.